.github/workflows/build-reproducer-container.yml

```diff
@@ -20,8 +27,22 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ github.token }}
         run: |
+          case "${{ matrix.arch }}" in
+            aarch64)
+              llvm_container_arch="arm64v8"
+              ;;
+            x86_64)
+              llvm_container_arch="amd64"
+              ;;
+          esac
+
+          if [ -n "$llvm_container_arch" ]; then
+            extra_args="--build-arg optimized_llvm=ghcr.io/llvm/$llvm_container_arch/ci-ubuntu-22.04:9f919661dd2f --build-arg llvm_sysroot=/opt/llvm/"
+          fi
+
           sudo apt-get -y update
-          sudo apt-get -y install podman
-          podman build -t "ghcr.io/$GITHUB_REPOSITORY_OWNER/llvm-snapshots-reproducer" -f Containerfile.mass-rebuild scripts/
+          sudo apt-get -y install podman qemu-user-static
+          # shellcheck disable=SC2086
+          podman build  --arch ${{matrix.arch }} --build-arg arch=${{ matrix.arch }} $extra_args -t "ghcr.io/$GITHUB_REPOSITORY_OWNER/llvm-snapshots-reproducer:${{ matrix.arch }}" -f Containerfile.mass-rebuild scripts/
```

Comment:
I learned to use `--platform` instead of `--arch` because it also sets other information, including `os` and `variant`.
I've already seen some weird cases where the environment in the container got mixed up.

Reply:
Thanks for the tip about `--platform`` that worked well and simplified the containerfile.